presto-verifier/src/main/java/com/facebook/presto/verifier/QueryRewriter.java

```diff
@@ -144,6 +169,18 @@ private String createTemporaryTableName()
         return rewritePrefix.getSuffix() + UUID.randomUUID().toString().replace("-", "");
     }
 
+    private List<Column> getColumnsForTable(Connection connection, String catalog, String schema, String table) throws SQLException
+    {
+        ResultSet columns = connection.getMetaData().getColumns(catalog, escapeLikeExpression(connection, schema), escapeLikeExpression(connection, table), null);
+        ImmutableList.Builder<Column> columnBuilder = new ImmutableList.Builder<>();
+        while (columns.next()) {
+            String name = columns.getString(4);
```

Comment:
Using named columns instead of indices, would be probably more readable. The names are documented e.g. @ https://docs.oracle.com/javase/7/docs/api/java/sql/DatabaseMetaData.html#getColumns(java.lang.String,%20java.lang.String,%20java.lang.String,%20java.lang.String)